examples/misc_controls_deviceorientation.html

```diff
@@ -13,13 +13,30 @@
 			photo by <a href="http://www.flickr.com/photos/jonragnarsson/2294472375/" target="_blank" rel="noopener">Jón Ragnarsson</a>.
 		</div>
 
+		<button id="deviceBtn">Open device orientation(Only in IOS 13+)</button>
```

Comment:
Actually, it's no good approach to show optional UI elements by default and hide them via JavaScript. When you refresh the page, you can see right at the beginning how the button is displayed and then disappears. The location at the top left is not optimal, too. 

I suggest you develop the code similar to [webgl_materials_video](https://threejs.org/examples/webgl_materials_video) and always request a user interaction at the beginning (with the text `Using device orientation might require user interaction.`).